src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.impl.CarServiceImpl;
+import mate.jdbc.service.impl.DriverServiceImpl;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+    private static DriverService driverService =
+            (DriverServiceImpl) injector.getInstance(DriverService.class);
+    private static CarService carService = (CarServiceImpl) injector.getInstance(CarService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        Car car = new Car("RenoA2",new Manufacturer(3L,"reno", "France"));
```

Comment:
I think it's not a good idea to set id manually, after creation your objects have id